src/Fixer/Operator/BinaryOperatorSpacesFixer.php

```diff
@@ -219,15 +354,423 @@ private function isDeclareStatement(Tokens $tokens, $index)
         return false;
     }
 
-    private function runHelperFixers(\SplFileInfo $file, Tokens $tokens)
+    private function resolveConfig(array $configuration)
+    {
+        $this->configuration = [];
```

Comment:
Later this should all be easy I think, as currently this is setup for some BC layer of the old configuration items.
I tried other ways but didn't work out. So if the configuration should be handled in another way please PR it because I don't know how the current-configuration-PR-way fits better in the new configuration system (yes, this PR pre-dates that X| )

Reply:
The way you handle the configuration is fine, I don't think there is a better way to handle cases like deprecated options at the moment anyway. But thinking about it again, the `$configuration` property is internal so current state is fine. If one needs to change that to add a new option, it should not be that hard actually so feel free to ignore my previous comment :)

Reply:
I raised concern for exactly same line as well: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/2573#discussion_r139283693 .
For me, we shall not change `configuration` property definition structure defined in parent class. Dev not aware of this fixer won't know that this property is handled differently than how it's defined by parent class.